src/ui/units/workbooks/components/WorkbookActions/WorkbookActions.tsx

```diff
@@ -62,117 +59,91 @@ export const WorkbookActions: React.FC<Props> = ({workbook, refreshWorkbookInfo}
+                                workbookTitle: workbook.title,
+                                initialCollectionId: workbook.collectionId,
+                                onApply: onApplyCopy,
+                                onClose: () => {
+                                    dispatch(closeDialog());
+                                },
+                            },
+                        }),
+                    );
+                },
+                text: i18n('action_copy'),
+            });
+        }
+    }
+
     return (
         <div className={b()}>
-            {collectionsAccessEnabled && workbook.permissions.listAccessBindings && (
+            {workbook.permissions.update && (
                 <div className={b('item')}>
-                    <Button
-                        onClick={() => {
-                            setIamAccessDialogIsOpen(true);
-                        }}
-                    >
-                        {i18n('action_access')}
-                    </Button>
+                    <CreateEntry view="action" />
                 </div>
             )}
-            {additionalActions.map((action) => {
-                return (
-                    <div key={action.id} className={b('item')}>
-                        <Button onClick={action.action}>{action.text}</Button>
-                    </div>
-                );
-            })}
 
-            {workbook.permissions.move && (
-                <div className={b('item')}>
-                    <Button
-                        onClick={() => {
-                            dispatch(
-                                openDialog({
-                                    id: DIALOG_MOVE_WORKBOOK,
-                                    props: {
-                                        open: true,
-                                        workbookId: workbook.workbookId,
-                                        workbookTitle: workbook.title,
-                                        initialCollectionId: workbook.collectionId,
-                                        onApply: refreshWorkbookInfo,
-                                        onClose: () => {
-                                            dispatch(closeDialog());
-                                        },
-                                    },
-                                }),
-                            );
-                        }}
-                    >
-                        {i18n('action_move')}
-                    </Button>
-                </div>
+            {Boolean(dropdownActions.length) && (
+                <DropdownMenu
+                    defaultSwitcherProps={{view: 'normal'}}
+                    switcherWrapperClassName={b('item')}
+                    items={dropdownActions}
+                />
             )}
 
-            {workbook.permissions.copy && (
+            {collectionsAccessEnabled && workbook.permissions.listAccessBindings && (
```

Comment:
1) Incorrect order, should be – "Create", "Access" and "...".
2) There are no icons, but they are in the design.

<img width="341" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/287567e5-c5ea-473b-9137-199a10530b8c">

<img width="221" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/a232bbe6-07f0-4108-8692-2d5eb9e27eb7">

Reply:
fixed

Reply:
Sorry, but in the latest version of the design the order should be like this:
<img width="284" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/8e0c9e5c-cb9e-47da-a77a-74888cc9179e">